streams/src/main/java/org/apache/kafka/streams/kstream/Consumed.java

```diff
@@ -70,6 +74,18 @@ private Consumed(final Serde<K> keySerde,
         this.processorName = processorName;
     }
 
+    private Consumed(final Serde<K> keySerde,
```

Comment:
We should not add a second constructor, but extend the existing one to take an additional parameter for the new AutoOffsetReset class

Reply:
added this, thank you